src/main/java/seedu/address/logic/parser/DateParser.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.logic.parser;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Represents a Parser that is able to parse Date.
+ */
+public class DateParser {
+    public static final String DATE_INVALID_FORMAT = "Date format should be either MM/DD/YYYY or MM/DD/YYYY HH:mm."
```

Comment:
Don't need to modify for now but we'll need to standardize on a specific date pattern for both deliverable and meeting. (Deliverable uses DD-MM-YYYY HH:MM format instead)